src/container/srv_target.c

```diff
@@ -509,6 +510,21 @@ cont_child_destroy_one(void *vin)
 	 * synchronous GC for now.
 	 */
 	dss_gc_run(-1);
+	/*
+	 * Force VEA to expire all the just freed extents and make them
+	 * available for allocation immediately.
+	 */
+	rc = vos_pool_query(pool->spc_hdl, &pinfo);
```

Comment:
Since I've added vos_pool_ctl, is it a better option than this way?

Reply:
You mean add a new vos_pool_ctl option and call vos_pool_query() in vos_pool_ctl()? Or do you think we don't have to force sync VEA on pool query and should provide another new VEA API to force sync?

Reply:
I think you can add a new opcode for vos_pool_ctl